jenkins/aws-janitor/Makefile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION = 0.0
+
+image:
+	CGO_ENABLED=0 go build k8s.io/test-infra/jenkins/aws-janitor
+	docker build -t "b.gcr.io/k8s-test-infra-aws/aws-janitor:$(VERSION)" .
+
+push: image
+	gcloud docker -- push "b.gcr.io/k8s-test-infra-aws/aws-janitor:$(VERSION)"
```

Comment:
@justinsb: I made a minor change to shift this to a bucket that you could push to in case something goes haywire. We can add other people later if we need to.